algrtm/src/components/OfferSection.js

```diff
@@ -17,14 +17,14 @@ const OfferSection = () => {
   };
 
   return (
-    <section className="flex flex-col w-full mt-[8.5rem] gap-[2.75rem] justify-center items-center">
-      <h2 className="flex h2 text-neutralLight-neutral10 leading-[4rem]">
+    <section className="flex flex-col w-full md:mt-[8.5rem] gap-[2.75rem] justify-center items-center">
+      <h2 className="flex h4 xl:h2 md:h2S text-neutralLight-neutral10">
         What we offer?
       </h2>
-      <div className="flex flex-col gap-4 w-full px-[5.875rem]">
+      <div className="flex flex-col gap-4 w-full xl:px-[5.875rem]">
         {/* RESEARCH & DISCOVERY */}
-        <div className="flex gap-[1.5rem]  bg-neutralLight-neutral90 py-[5.85rem] px-[6.75rem] rounded-[2rem]">
-          <div className="h4 text-neutralLight-neutral10 leading-[2.75rem] flex justify-start max-w-[25.1875rem] w-full">
+        <div className="flex flex-col md:flex-row gap-[1.5rem]  bg-neutralLight-neutral90 px-[2.75rem] py-[2.75rem] md:py-[5.88rem]  xl:py-[5.85rem] xl:px-[6.75rem] rounded-[2rem]">
+          <div className="h4SB md:h4 text-neutralLight-neutral10 leading-[2.75rem] flex justify-start max-w-[25.1875rem] w-full">
```

Comment:
According to Figma, the font-size for small devices should be 28px, for medium 32px, and for large 44px. Currently we have 32px for small and 44px for medium and large